pkg/config/deployment.go

```diff
@@ -381,3 +383,27 @@ func (e *SecretEncryption) Validate() error {
 	}
 	return nil
 }
+
+// DeploymentNotification represents the way to send to users.
+type DeploymentNotification struct {
+	// List of users to be notified for each event.
+	Mentions []NotificationMention `json:"mentions"`
+}
+
+type NotificationMention struct {
+	// The event to be notified to users.
+	Event string `json:"event"`
+	// The user name for mentioning in Slack.
+	Slack []string `json:"slack"`
+	// The email for notification.
+	Email []string `json:"email"`
+}
+
+func (n *NotificationMention) Validate() error {
+	for k := range model.NotificationEventType_value {
+		if n.Event == k {
+			return nil
+		}
+	}
+	return fmt.Errorf("the field name %s must be NotificationEventType", n.Event)
```

Comment:
Nice to have the field name like:
```suggestion
	return fmt.Errorf("event %q is incorrect as NotificationEventType", n.Event)
```